todo_app/app.py

```diff
@@ -10,8 +10,8 @@ def create_app():
 
     @app.route('/')
     def index():
-        items = get_items()
-        item_view_model = ViewModel(items)
+        allCards = get_items()
```

Comment:
I'm suspicious that this didn't come with a change to an import (suggests no file name change, see elsewhere) but otherwise the fact that this is all that needs to change in this file is a really good sign that we picked sensible levels of function previously, since we can swap out our data persistence without this or the HTML having to change - nice! 👍